app/Services/OpeninghoursService.php

```diff
@@ -0,0 +1,274 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\Channel;
+use App\Models\Service;
+use Carbon\Carbon;
+
+/**
+ *
+ */
+class OpeninghoursService
```

Comment:
Check out the https://murze.be/2016/10/managing-opening-hours-php/ package and see if we can use it.